doc/decisions/STEPS.md

```diff
@@ -54,28 +78,34 @@ Here you must ensure:
 Here the decision should not only have one decision but should describe several solutions.
 For each solution a proposal, rationale and optionally implications should be given.
 
-## In Progress
+## In Review
 
 - You must include all further alternative proposals made in the "Considered Alternatives" section.
 - Now it is allowed to have the decision from the previous round in the "Decision" section.
 
-> For decisions "In Progress", unlike "In Discussion", also the solution space next to the problem is clear.
-> It is the highest of the steps reachable for a decision PR that started in "Drafts".
+> For decisions "In Review" the solution space is clear.
+> I.e. the trade-offs, combinations and pros/cons of the considered alternatives are explored.
+> Decision author and reviewers are satisfied that every useful considered alternative is present in the decision.
+> Without merges in between, this is the last step reachable for a decision PR that started in "Drafts".
 
 ## Decided
 
 > This step is mandatory.
-> I.e., there must be a decision PR that puts the decision into "Decided".
+> I.e., there must be a dedicated decision PR that puts the decision into "Decided".
 
 - "Decision", "Rationale" and "Implications" are now filled out and fixed according to the reviews
 - decisions of this step usually already have an implementation PR
 
+Decisions that need an update, e.g. because assumptions changed, usually directly start with the step "Decided".
```

Comment:
I think that needs to be phrase differently. IMO it should only start in "Decided", if the chosen solution stays the same and only details change. If the problem it self changes, or the chosen solution is no longer a good choice, then we should probably start in an earlier step.

Reply:
Changed usually -> sometimes. I don't think we should put rules on rare cases. e447fde4d313e45aafc416f7073a520845dc8dba